pkg/sdk/testint/streams_gen_integration_test.go

```diff
@@ -0,0 +1,352 @@
+package testint
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInt_Streams(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	db, cleanupDb := createDatabase(t, client)
+	t.Cleanup(cleanupDb)
+
+	schema, cleanupSchema := createSchema(t, client, db)
+	t.Cleanup(cleanupSchema)
+
+	t.Run("CreateOnTable", func(t *testing.T) {
+		table, cleanupTable := createTable(t, client, db, schema)
+		t.Cleanup(cleanupTable)
+
+		id := sdk.NewSchemaObjectIdentifier(db.Name, schema.Name, randomAlphanumericN(t, 32))
+		req := sdk.NewCreateOnTableStreamRequest(id, table.ID()).WithComment(sdk.String("some comment"))
+		err := client.Streams.CreateOnTable(ctx, req)
+		require.NoError(t, err)
+		t.Cleanup(func() {
+			err := client.Streams.Drop(ctx, sdk.NewDropStreamRequest(id))
+			require.NoError(t, err)
+		})
+
+		s, err := client.Streams.ShowByID(ctx, sdk.NewShowByIdStreamRequest(id))
+		require.NoError(t, err)
```

Comment:
can we check the whole set of values?

Reply:
And also, because they are validated in a few tests, can we extract some helper assertions? like in task/database_role

Reply:
Done